src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -370,4 +383,35 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
 
         onZoomChanged(zoom);
     };
+
+    private handleMapClick = (e: mapboxgl.EventData) => {
+        const {
+            config: { viewport },
+            drillConfig,
+        } = this.props;
+        const { features, originalEvent, target } = e;
+        const { properties } = features[0];
+
+        // Disable drilling in edit/export mode
+        if (get(viewport, "frozen", false)) {
+            return;
+        }
+
+        // without this, the drill dialog will be closed
+        originalEvent.stopPropagation();
+
+        return handleGeoPushpinDrillEvent(drillConfig, properties, target);
```

Comment:
We should have new `getDrillProps` func, like
```
...
return handleGeoPushpinDrillEvent(drillConfig, getDrillProps(properties), target);
```

I mean we should pass down what we need

Reply:
updated